src/modules/MeasureTool/MeasureToolCore/MeasureToolOverlayUI.cpp

```diff
@@ -32,8 +32,6 @@ namespace
 winrt::com_ptr<ID2D1Bitmap> ConvertID3D11Texture2DToD2D1Bitmap(winrt::com_ptr<ID2D1RenderTarget> rt,
                                                                const MappedTextureView* capturedScreenTexture)
 {
-    capturedScreenTexture->view.pixels;
```

Comment:
Is this really safe to remove? Was this not there to force a read to initialize something? This was your original code, right @yuyoyuppe ?